packages/fxa-settings/src/components/Settings/MfaGuard/index.tsx

```diff
@@ -71,30 +71,26 @@ export const MfaGuard = ({
   // Modal Setup
   useEffect(() => {
     (async () => {
-      // To avoid requesting multiple OTPs on mount
+      const hasJwt = JwtTokenCache.hasToken(sessionToken, requiredScope);
```

Comment:
Was there a bug here?

Reply:
Yes, the modal was not closing after confirming the mfa code

Reply:
I've extracted this into a separate PR: https://github.com/mozilla/fxa/pull/19424